framework/src/org/checkerframework/framework/type/GenericAnnotatedTypeFactory.java

```diff
@@ -1275,15 +1274,8 @@ protected void addComputedTypeAnnotations(
         defaults.annotate(tree, type);
 
         if (iUseFlow) {
-            Value as;
-            if (tree.getKind() == Kind.POSTFIX_DECREMENT
-                    || tree.getKind() == Kind.POSTFIX_INCREMENT) {
-                // Dataflow incorrectly treats postfix as prefix.
-                // See Issue 867: https://github.com/typetools/checker-framework/issues/867
-                as = getInferredValueFor(((UnaryTree) tree).getExpression());
```

Comment:
[BaseTypeVisitor](https://github.com/typetools/checker-framework/blob/master/framework/src/org/checkerframework/common/basetype/BaseTypeVisitor.java#L1343) references this work around.  Now that the work around is removed, the comment in BaseTypeVisitor can also be removed.